src/include/cell.i

```diff
@@ -329,6 +350,10 @@ __wt_cell_pack_value_match(
         if (validity) { /* Skip validity window */
             flags = *b;
             ++b;
+            if (LF_ISSET(WT_CELL_TS_DURABLE_START))
+                WT_RET(__wt_vunpack_uint(&b, 0, &v));
+            if (LF_ISSET(WT_CELL_TS_DURABLE_STOP))
+                WT_RET(__wt_vunpack_uint(&b, 0, &v));
```

Comment:
I've been thinking for awhile that maybe we should wrap this whole group in:
```
if (flags != 0) {
        all of the flag tests
}
```
There are like 4 places in `cell.i` where we step through a long set of tests, and I think (emphasis on "think"), we generally expect `flags == 0`. Unpacking cells is an big part of WT's performance, so it's worth a thought.

Reply:
It feels right.  I don't know for certain that `flags == 0` is the usual case.  I'll try to find out.  Anyway, I've made this change for now.

Reply:
Now I think this is not really an optimization.  The only reason to use the extended byte is if one of the bits is set, right?  So we should never see `flags == 0`.

Reply:
@keithbostic, I really liked the `if (flags != 0)` idea, but I don't think it works out.  It looks like if all of the bits are 0, then the extended byte won't even be present in the file.  Does that sound correct?

Reply:
> It looks like if all of the bits are 0, then the extended byte won't even be present in the file.

Ack, I think you're correct.